CONTRIBUTING.md

```diff
@@ -111,6 +170,15 @@ crates/meroctl/src/cli/app/list.rs
 
 ### Error Handling
 
+- We use the eyre crate extensively in our code, and import it as follows:
+  
+  ```rust
+  use eyre::{Result as EyreResult};
+  ```
+
+- Almost no unwrapping (acceptable in tests and possibly when dealing with thread join handlers).
+- If unwrapping is absolutely necessary, explain why with a comment.
```

Comment:
it's never necessary, just that the author has a lot of confidence in it never panicking